jbpm-wb-forms/jbpm-wb-forms-client/src/main/java/org/jbpm/workbench/forms/client/display/process/QuickNewProcessInstancePopup.java

```diff
@@ -168,8 +169,8 @@ protected void loadFormValues(String serverTemplateId) {
             Scheduler.get().scheduleDeferred(() -> processDefinitionsListBox.refresh());
         }).getProcesses(serverTemplateId,
                         0,
-                        1000,
-                        "",
+                        Integer.MAX_VALUE,
```

Comment:
@nmirasch just curious, the fix on the 6.5.x had a -1 :)

Reply:
This line is the guilty line https://github.com/kiegroup/jbpm-wb/blob/5800896515ea40d201600f8174a6eeeb99b790a9/jbpm-console-ng-process-runtime/jbpm-console-ng-process-runtime-backend/src/main/java/org/jbpm/console/ng/pr/backend/server/ProcessDefinitionServiceImpl.java#L73-L73. At 6.5.x version the service getProcessDefinitions(final QueryFilter filter) adds 1 to the page size.

Reply:
that explains :)